KmsClient.js

```diff
@@ -209,7 +222,7 @@ export class KmsClient {
     _assert(invocationSigner, 'invocationSigner', 'object');
 
     const operation = {
-      '@context': securityContextUrl,
+      '@context': [WEBKMS_CONTEXT_URL],
```

Comment:
Why not just `WEBKMS_CONTEXT_URL`? Why the array? Let's not use an array unless we really need one.

```suggestion
      '@context': [WEBKMS_CONTEXT_URL],
```

Reply:
addressed here 049942d